sys/net/application_layer/gcoap/gcoap.c

```diff
@@ -139,7 +139,7 @@ static void _on_sock_evt(sock_udp_t *sock, sock_async_flags_t type)
         ssize_t res = sock_udp_recv(sock, _listen_buf, sizeof(_listen_buf),
                                     0, &remote);
         if (res <= 0) {
-            DEBUG("gcoap: udp recv failure: %d\n", res);
+            DEBUG("gcoap: udp recv failure: %d\n", (int)res);
```

Comment:
This issue is the usual problem with an `ssize_t` type on pic32. You can see a similar cast just below.

Previously, this DEBUG statement was within #if macro that was always false in the CI. In other words, it's a bug fix. :-)